ml-agents/mlagents/trainers/policy/policy.py

```diff
@@ -32,23 +32,18 @@ def __init__(
         self.trainer_settings = trainer_settings
         self.network_settings: NetworkSettings = trainer_settings.network_settings
         self.seed = seed
-        self.act_size = (
-            list(self.behavior_spec.action_spec.discrete_branches)
-            if self.behavior_spec.action_spec.is_discrete()
-            else [self.behavior_spec.action_spec.continuous_size]
-        )
-        self.vec_obs_size = sum(
-            sen_spec.shape[0]
-            for sen_spec in behavior_spec.sensor_specs
-            if len(sen_spec.shape) == 1
-        )
-        self.vis_obs_size = sum(
-            1 for sen_spec in behavior_spec.sensor_specs if len(sen_spec.shape) == 3
-        )
-        self.use_continuous_act = self.behavior_spec.action_spec.is_continuous()
         self.previous_action_dict: Dict[str, np.ndarray] = {}
         self.memory_dict: Dict[str, np.ndarray] = {}
         self.normalize = trainer_settings.network_settings.normalize
+        if self.normalize:
```

Comment:
What happens if self.normalize is True but none of them are vector obs? I.e. can we just remove this flow?

Reply:
We can remove it, I just thought this was cleaner, but it does add some complexity.

Reply:
Removed